Tasks/AppStoreRelease/task.loc.json

```diff
@@ -15,7 +15,7 @@
   ],
   "version": {
     "Major": "1",
-    "Minor": "189",
+    "Minor": "194",
```

Comment:
Could you please bump the sprint number to 203 (per [calendar](https://whatsprintis.it/))?